modules/hdf5/src/processors/hdf5volumesource.cpp

```diff
@@ -62,7 +62,7 @@ HDF5ToVolume::HDF5ToVolume()
 
     , basisGroup_("basisGroup", "Basis")
     , basisSelection_("basisSelection", "Source")
-    , basis_("basis", "Matrix", mat4(1.0f), mat4(-10.0f), mat4(10.0))
+    , basis_("basis", "Matrix", mat4(1.0f), mat4(-1000.0f), mat4(1000.0))
```

Comment:
this will make min and max be diagonal matrices, use `util::filled<mat4>(1000.f)` (https://github.com/inviwo/inviwo/blob/master/include/inviwo/core/util/glm.h#L815-L828)